document-service/documents-ui/src/components/documentRecord/EditScanning.vue

```diff
@@ -50,12 +54,13 @@ defineProps({
             />
           </div>
         </div>
-
-        <InputDatePicker
-          v-model="scanningDetails.scanDateTime"
-          :date-placeholder="$t('scanningDetails.scannedDate')"
+        <UInput
+          v-model="scannedDate"
+          :placeholder="$t('scanningDetails.scannedDate')"
           class="mt-3"
-          :is-trailing="true"
+          :disabled="!hasScanningDetails"
+          readonly
+          :ui="{ base: 'border-b-[2px] border-dotted'}"
```

Comment:
Don't we still need a Date Picker for when enabled?

Reply:
As I know this field is readonly when we have scanning details.

Reply:
Oh ok, so it's never Editable?
The way i interpret the ticket is: 
Readonly **when** we have scanning details. 
Question for Liz perhaps, still editable when we **don't** have scanning details?

Reply:
Interesting question: the scanning details come from the scanning app so should they be editable in the DRS UI?